docs/documentation/getting-started/deployment/manual/platform-setup.md

```diff
@@ -175,6 +192,34 @@ You can switch between the clusters easily using:
                     name: ephemeral-mpc-engine-port-3
                   - port: 30004
                     name: ephemeral-mpc-engine-port-4
+        egressGateways:
+          - name: istio-egressgateway
+            enabled: true
+            k8s:
+              resources:
+                requests:
+                  cpu: 10m
```

Comment:
Do we know that this is enough? Assuming that the traffic from Epehemal and Klyshko goes over the egress gateway, it might be too less.

Reply:
No we don't. We can for sure increase it, but it for now all values would be guessing - at least for me